### PR TITLE
MainSettings: Make MAIN_CUSTOM_RTC_VALUE default value less magical

### PR DESCRIPTION
**Pull request by Pokechu22**
Suggested in https://github.com/dolphin-emu/dolphin/pull/8067#discussion_r878611999.

### CODE REVIEW

File: Source/Core/Core/Config/MainSettings.cpp [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 #include "Core/Config/MainSettings.h"
 
+#include <chrono>
```
Comment (iwubcode) [inline Source/Core/Core/Config/MainSettings.cpp:6]: It really bugs me that I can't write any chrono that is nicer than all those multiplies but even with MSVC's C++20, it's still quite verbose.

Anyway, I think you can remove this.

Comment (Pokechu22) [reply Source/Core/Core/Config/MainSettings.cpp:6]: Ah, no, I was trying to implement it using std::chrono to do the date differences but I couldn't get it to work; that's a leftover from my attempt.

Comment (iwubcode) [reply Source/Core/Core/Config/MainSettings.cpp]: `chrono` should be possible but it's a lot more verbose, even with C++20  :(

**Comment by AdmiralCurtiss** [comment]
Sure, why not.

**Pull request merged**